src/Veldrid.StartupUtilities/Veldrid.StartupUtilities.csproj

```diff
@@ -16,4 +16,12 @@
     <PackageTags>Core Standard Game SDL2 Window Input</PackageTags>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DefineConstants>TRACE;DEBUG;GFXAPI_VULKAN_ENABLED;NETSTANDARD2_0</DefineConstants>
```

Comment:
These shouldn't be defined here. Just add this line tot he first `PropertyGroup` of the file:

`<DefineConstants>$(DefineConstants);GFXAPI_VULKAN_ENABLED</DefineConstants>